plugins/source/aws/docs/tables/aws_support_trusted_advisor_check_results.md

```diff
@@ -0,0 +1,25 @@
+# Table: aws_support_trusted_advisor_check_results
+
+https://docs.aws.amazon.com/awssupport/latest/APIReference/API_DescribeTrustedAdvisorCheckResult.html
+
+The primary key for this table is **_cq_id**.
+
+## Relations
+
+This table depends on [aws_support_trusted_advisor_checks](aws_support_trusted_advisor_checks.md).
+
+## Columns
+
+| Name          | Type          |
+| ------------- | ------------- |
+|_cq_source_name|String|
+|_cq_sync_time|Timestamp|
+|_cq_id (PK)|UUID|
+|_cq_parent_id|UUID|
+|account_id|String|
+|category_specific_summary|JSON|
+|check_id|String|
```

Comment:
Could `check_id` serve as a primary key here?

Reply:
Yes! Good catch, I missed that (thought we get a results array). Fixed in https://github.com/cloudquery/cloudquery/pull/8140/commits/1a10e80ef72917c3b0f3528b58fe838ea5df5da7